presto-docs/src/main/sphinx/sql/grant.rst

```diff
@@ -0,0 +1,51 @@
+=====
+GRANT
+=====
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    GRANT ( privilege [, ...] | ( ALL PRIVILEGES ) )
+    ON [ TABLE ] table_name TO ( grantee | PUBLIC )
+    [ WITH GRANT OPTION ]
+
+Usage of the term ``grantee`` denotes both users and roles.
+
+Description
+-----------
+
+Grants the specified privileges to the specified grantee.
+
+Specifying ``ALL PRIVILEGES`` grants ``DELETE, INSERT`` and ``SELECT`` privileges.
+
+Specifying ``PUBLIC`` grants privileges to all grantees.
+
+The optional ``WITH GRANT OPTION`` clause allows the grantee to grant these same privileges to others.
+
+For ``Grant`` statement to succeed, the user executing it should possess the specified privileges as well as the ``GRANT OPTION`` for those privileges.
+
+Limitation
+----------
+
+``GRANT`` is currently supported only for hive connector.
```

Comment:
I see. Ok, I will add the following line in the "Limitations" section here: 
```Many connectors have no support for GRANT. See connector documentation for more details.```

And I will also modify each of the connector's documentation to add ```GRANT``` under limitations for that connector.